pkg/daemon/ceph/client/upgrade.go

```diff
@@ -400,3 +411,14 @@ func osdDoNothing(context *clusterd.Context, clusterInfo *ClusterInfo) bool {
 
 	return false
 }
+
+func getRetryConfig(clusterInfo *ClusterInfo, daemonType string) (int, time.Duration) {
+	switch daemonType {
+	case "osd":
+		return int(clusterInfo.OsdUpgradeTimeout / defaultOSDRetryDelay), defaultOSDRetryDelay
+	case "mds":
```

Comment:
```suggestion
	case cephv1.KeyMDS:
```

Reply:
You need to add it.

Reply:
Like discussed yesterday, not adding keyType constants in this PR as it would requiring changing the `UpdateCephDeploymentAndWait` signature in a lot of places.